### PR TITLE
Resolves #72 Gives Ad Hocs Their Own Section (Again)

### LINKED ISSUE #72: Give adhocs their own section
Check one:
- [x] Semantic Issue: something about the meaning of the text is different
- [ ] Non-semantic Issue: Spelling, grammar, or formatting changes.

Summary of issue:
Adhocs are currently in the eboard section, but adhocs work differently enough from permanent eboard positions that it deserves its own section.

### PR DESCRIPTION
**Pull request by Sam-Cordry**
Check one:
- [x] Semantic Change: something about the meaning of the text is different
- [ ] Non-semantic Change: Spelling, grammar, or formatting changes.

Summary of change(s):
Resolves #72 by adding in a section for Ad Hoc Directorships in Article 5 and refactoring some content around it (again).
Check #96 for previous discussion.

### CODE REVIEW

File: constitution.tex [hunk 1]
```diff
@@ -719,6 +703,21 @@ \subsubsection{Selection of the Chair, Evals Director, Social Director(s), Finan
 After the appeal is presented, an Executive Board Vote is taken to determine whether or not to overturn and reevaluate the decision.
 If the vote passes, the Executive Board may discuss and make a new ruling by another Executive Board Vote.
 
+\asection{Ad Hoc Directorships}
+Ad Hoc Directorships are created by the Executive Board on an as-needed basis.
+They are generally task-oriented and are chaired by a House member(s).
```
Comment (danisaba) [inline constitution.tex:708]: This and the line below about them being "active members" should go in a section titled "qualifications of adhoc..." just like the eboard positions to have CONSISTENCYYYYYYY RAAAHHH

File: constitution.tex [hunk 2]
```diff
@@ -719,6 +703,21 @@ \subsubsection{Selection of the Chair, Evals Director, Social Director(s), Finan
...
+Ad Hoc Directors are responsible for the task for which they were created.
+Additionally, any responsibilities, budgets, or finances for the Ad Hoc Directorship are placed upon the assigned director(s).
+Candidates for being Ad Hoc Directors must be Active Members.
+% This is implied, but Ad Hoc Directorships are not intended to be restricted to single or dual directorship rules.
+
+\asubsection{Selection of Ad Hoc Directors}
+\begin{enumerate}
+	\item When the Executive Board or a group of House members feels an Ad Hoc Directorship is necessary, they present their plans to the Executive Board.
+	An Executive Board Vote is taken to determine whether directorship status is granted.
+	\item When the Ad Hoc Directorship is granted status, a director is appointed, and duties, budgetary, and membership considerations are defined.
```
Comment (danisaba) [inline constitution.tex:718]: make two sentences. I don't like the word budgetary :pleading_face:

File: constitution.tex [hunk 3]
```diff
@@ -719,6 +703,21 @@ \subsubsection{Selection of the Chair, Evals Director, Social Director(s), Finan
 After the appeal is presented, an Executive Board Vote is taken to determine whether or not to overturn and reevaluate the decision.
 If the vote passes, the Executive Board may discuss and make a new ruling by another Executive Board Vote.
 
+\asection{Ad Hoc Directorships}
+Ad Hoc Directorships are created by the Executive Board on an as-needed basis.
+They are generally task-oriented and are chaired by a House member(s).
+Ad Hoc Directors are responsible for the task for which they were created.
+Additionally, any responsibilities, budgets, or finances for the Ad Hoc Directorship are placed upon the assigned director(s).
+Candidates for being Ad Hoc Directors must be Active Members.
```
Comment (danisaba) [inline constitution.tex:711]: see above comment

**Review by danisaba — CHANGES_REQUESTED** [review]
change it. (love u sam)

File: constitution.tex [hunk 4]
```diff
@@ -719,6 +703,21 @@ \subsubsection{Selection of the Chair, Evals Director, Social Director(s), Finan
...
+% This is implied, but Ad Hoc Directorships are not intended to be restricted to single or dual directorship rules.
```
Comment (rhochgraf21) [inline constitution.tex:712]: In my opinion, this is best said explicitly

**Comment by esoccoli** [comment]
Decided to pursue this in a separate amendment later

**Pull request closed without merging**